packages/dx-react-grid-bootstrap3/src/templates/table-fixed-cell.test.jsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { shallow, mount } from 'enzyme';
+import { FixedCell } from './table-fixed-cell';
+import { ThemeColors } from './layout';
+
+const defaultProps = {
+  column: { name: 'Test' },
+  side: 'left',
+  component: () => <span />,
+};
+const themeColors = {
+  backgroundColor: 'red',
+  borderColor: 'green',
+};
+
+describe('FixedCell', () => {
+  it('should apply default style', () => {
+    const tree = shallow((
+      <FixedCell {...defaultProps} />
+    ));
+
+    expect(tree.prop('style')).toMatchObject({
+      position: 'sticky',
+      backgroundClip: 'padding-box',
+      zIndex: 300,
+    });
+  });
+
+  it('should apply left border if left divider exists', () => {
+    const tree = mount((
+      <ThemeColors.Provider value={themeColors}>
+        <FixedCell {...defaultProps} showLeftDivider />
+      </ThemeColors.Provider>
+    ));
+
+    expect(tree.childAt(0).prop('style')).toMatchObject({
+      borderLeft: '1px solid green',
+    });
+  });
+
+  it('should apply right border if right divider exists', () => {
+    const tree = mount((
+      <ThemeColors.Provider value={themeColors}>
+        <FixedCell {...defaultProps} showRightDivider />
+      </ThemeColors.Provider>
+    ));
+
+    expect(tree.childAt(0).prop('style')).toMatchObject({
+      borderRight: '1px solid green',
+    });
+  });
+
+  it('should pass custom styles to the root element', () => {
+    const tree = shallow((
+      <FixedCell {...defaultProps} style={{ color: 'white' }} />
+    ));
+
+    expect(tree.prop('style')).toMatchObject({
+      color: 'white',
```

Comment:
I'd check `{ color: 'white' }` doesn't completely override styles but extends them. E.g. check that the `position: sticky` exists too.